libs/cli/langchain_cli/project_template/README.md

```diff
@@ -1,11 +1,11 @@
-# LangServeHub Project Template
+# LangServe App Template
```

Comment:
there's nothing in here aboutt starting the app? imo we should add that, and then add in a section before that about setting up langsmith